pyedflib/tests/test_edfreader.py

```diff
@@ -98,24 +108,24 @@ def test_EdfReader_BDF_datarec_0_5(self):
         np.testing.assert_equal(f.datarecords_in_file, datarecords / datarecord_duration)
         np.testing.assert_equal(f.getFileDuration(), datarecords)
         
-        sample_frequencies = [1000, 800, 500, 975, 999]
+        sample_frequencies = [2000, 1600, 1000, 1950, 1998]
 
```

Comment:
these were wrong before. Now they are correct and finally are also what EDFBrowser would display.